Resources/Prototypes/Entities/Objects/Misc/books.yml

```diff
@@ -268,105 +357,173 @@
+      - cover:
+          cover_base: Sixteen
+          cover_old: Sixteen
+          cover_strong: Sixteen
+        decor:
+          decor_wingette: Sixteen
+          decor_wingette_circle: Sixteen
+          decor_bottom: Sixteen
+          decor_middle: Sixteen
+          decor_spine: Sixteen
+          decor_diagonal: Sixteen
+          decor_vertical_middle: Sixteen
+          icon_corner: Sixteen
+          icon_mount: ""
+        icon:
+          icon_biohazard: Sixteen
+          icon_borg: ""
+          icon_banana: ""
+          icon_glow: Sixteen
+          icon_hacking: ""
+          icon_law: Sixteen
+          icon_magnifier: ""
+          icon_nuclear: ""
+          icon_time: Sixteen
+          icon_aurora: Sixteen
+          icon_briefcase: ""
+          icon_eye: ""
+          icon_letter_N: ""
+          icon_letter_P: ""
+          icon_lightning: ""
+          icon_planet: ""
+          icon_possum: ""
+          icon_question: Sixteen
+          icon_scmmd: ""
+          icon_stars: Sixteen
+          icon_stars2: Sixteen
+          icon_temple: Sixteen
+          icon_tree: ""
+          icon_pentagramm: Sixteen
+          icon_fish: ""
+          icon_origami: ""
+          icon_skull: ""
+          icon_text: ""
+          icon_text2: ""
+          icon_text3: "" 
+        overlay:
+          overlay_blood: ""
+          overlay_dirt: Sixteen
+          detail_bookmark: Sixteen
+          detail_rivets: Sixteen
+          overlay_null: ""
 
 - type: entity
-  parent: BookBase
-  id: BookEscalation
-  name: Robert's Rules of Escalation
-  description: The book is stained with blood. It seems to have been used more as a weapon than reading material.
-  components:
-    - type: Sprite
-      sprite: Objects/Misc/books.rsi
-      layers:
-        - state: book2
-    - type: MeleeWeapon
-      wideAnimationRotation: 180
-      damage:
-        types:
-          Blunt: 6
-    - type: Paper
-      content: book-text-escalation
-
-- type: entity
-  parent: BookBase
-  id: BookEscalationSecurity
-  name: "Robert's Rules of Escalation: Security Edition"
-  description: The book is stained with blood. It seems to have been used more as a weapon than reading material.
+  parent: BookRandom
+  id: BookRandomStory
+  suffix: random visual, random story
   components:
-    - type: Sprite
-      sprite: Objects/Misc/books.rsi
-      layers:
-        - state: book2
-    - type: MeleeWeapon
-      wideAnimationRotation: 180
-      damage:
-        types:
-          Blunt: 6
-    - type: Paper
-      content: book-text-escalation-security
-
+  - type: PaperRandomStory
+    storySegments:
+     # To do (TheShuEd): English is not my native language. Someone, make the text generation more correct and cool.
+    - "This is a"
+    - book_genre
+    - "about a"
+    - book_character_trait
+    - book_character
+    - and
+    - book_character_trait
+    - book_character
+    - "."
+    - "Due to"
+    - book_event
+    - ", they"
```

Comment:
The space separator is inserted before the comma here, causing some bad grammar. The same issue is present with every other dot above and below.